src/componets/ArticleCard.jsx

```diff
@@ -1,25 +1,20 @@
 import { Link } from "react-router-dom";
 
-export default function ArticleCard({ searchedArticles }) {
+export default function ArticleCard({ article }) {
   return (
-    <>
-      <section className="articles-list">
-        {searchedArticles.map((article) => {
-          return (
-            <section key={article.article_id}>
-              <ul className="article-card">
-                <li id="article-title">
-                  <Link to={`/articles/${article.article_id}`}>
-                    {article.title}
-                  </Link>
-                </li>
-                <li>{article.author}</li>
-                <li>{article.topic}</li>
-              </ul>
-            </section>
-          );
-        })}
-      </section>
-    </>
+    <div>
+      <ul className="article-card">
```

Comment:
This is nice but could be a component that way it could be re-used between all the component that use it

Reply:
I will do it now! Thank you!